observability/tools/loki.md

```diff
@@ -0,0 +1,39 @@
+# Loki
+
+Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system, created by Grafana
+Labs inspired by the learnings from Prometheus. Loki is commonly referred as Prometheus but for logs, which
+makes total sense once both tools follow the same architecture, which is a agent collecting metrics in each
+of the components of the software system, a server which stores the logs and also the Grafana dashboard, which
+access the loki server to build its visualizations and queries. That being said, Loki has three main
+components:
+
+## Promtail
+
+Its the agent portion of Loki. It can be used to grab metrics from several different places, like var/log/ for
+example. The configuration of the Promtail is a yaml file called ```config-promtail.yml``` commonly store on
+the  ```/usr/local/bin``` path. In this file, its described all the paths and log sources that will be
```

Comment:
Please double-check the pathing. `/usr/local/bin` is generally not where configuration files for system services go.

Reply:
Actually I just deleted reference to the path and it looks better! Thanks for calling that out!